apps/docker/Dockerfile

```diff
@@ -1,38 +1,37 @@
-# Librelingo is not compatible with Node 16, using Node 14.18.3 image instead
-FROM node:14.18.3 
+FROM archlinux:base-20220227.0.49015
```

Comment:
this way, we'll have to keep updating this a lot, no? is there a good mechanism to keep this up to date? at least with the node image, it's mainly about updating major node versions :thinking: (although I can see, we had a very specific tag there as well)

Reply:
A specific tag is used so that if anyone runs the dockerfile they will get the same image as you would get on docker hub. Using no tag could cause problems for some people running the dockerfile as the new arch image used might have changes that make other parts of the dockerfile not work

Reply:
Though, I might change the docker to use a distro with a package manager where I can specify specific version of packages, so that there won't be any problems when running the dockerfile when there are newer packages than the ones used in the image on docker hub

Reply:
My problem with the node image was the finicky way of having to setup pip and python 3

Reply:
I might use an alpine image as alpine images are small and it's package manager allows you to specify package version but alpine images/containers can be a bit hard to work with though

Reply:
hmmm, right, it makes sense to not use the `node` image, that's for sure.  yeah the specific tag is needed for sure, I mean we need to have a specific version of node and python

btw what about having 2 different dockerfiles for python stuff and node stuff? would that not make things much simpler?

Reply:
ok, using alpine seems more standard I think, so I like that idea